### PR TITLE
Update debug python button

### PR DESCRIPTION
**Pull request by paulacamargo25**
Closed: https://github.com/microsoft/vscode-python-debugger/issues/137

Screenshot 2023-11-21 at 5 08 11 PM

### CODE REVIEW

**Comment by roblourens** [comment]
What's the difference between "Debug Python File" and "Python Debugger: Debug Python File"?

File: src/extension/debugger/configuration/launch.json/launchJsonReader.ts [hunk 1]
```diff
@@ -5,13 +5,17 @@ import * as path from 'path';
 import * as fs from 'fs-extra';
 import { parse } from 'jsonc-parser';
 import { DebugConfiguration, Uri, WorkspaceFolder } from 'vscode';
-import { getWorkspaceFolder } from '../../../common/vscodeapi';
+import { getConfiguration, getWorkspaceFolder } from '../../../common/vscodeapi';
 
 export async function getConfigurationsForWorkspace(workspace: WorkspaceFolder): Promise<DebugConfiguration[]> {
```
Comment (roblourens) [inline src/extension/debugger/configuration/launch.json/launchJsonReader.ts:10]: What do you need to look up launch configs for?

**Review by roblourens — APPROVED** [review]

**Comment by paulacamargo25** [comment]
> What's the difference between "Debug Python File" and "Python Debugger: Debug Python File"?

The first one come from the Python Extension and the new one is for the Python Debugger. When we remove the debugger from the python debugger there will only be one.

File: src/extension/debugger/configuration/launch.json/launchJsonReader.ts (hunk 1 above)
Comment (paulacamargo25) [reply src/extension/debugger/configuration/launch.json/launchJsonReader.ts:10]: According to the proposal, we check if there is any configuration already created. If not, we show the user the prompt to create the launch.json

**Comment by roblourens** [comment]
I would suggest removing the "Python Debugger" prefix from this extension's commands, it seems redundant

**Pull request merged**